tests/integration/Travel/Handlers/Command/DuplicateTravelHandlerTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Travel\Handlers\Command;
+
+use Cake\Chronos\Date;
+use CommandHandlerTest;
+use Model\Travel\Command;
+use Model\Travel\Command\Travel;
+use Model\Travel\Commands\Command\DuplicateTravel;
+use Model\Travel\Passenger;
+use Model\Travel\Repositories\ICommandRepository;
+use Money\Money;
+use function end;
+
+class DuplicateTravelHandlerTest extends CommandHandlerTest
```

Comment:
Tohle stačí jako unit test v `Model\Travel\CommandTest` (prostě se testuje přímo ten agregát a není potřeba to ukládat do DB), ale klidně to můžu dodatečně upravit já.

Reply:
Nevím, jestli jsem to změnil tak, jak jsi myslel, teď by bylo potřeba použít mock na CommandRepository i Command